init-letsencrypt.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v docker-compose)" ]; then
+  echo 'Error: docker-compose is not installed.' >&2
+  exit 1
+fi
+
+domains=(osmintegrator.eu www.osmintegrator.eu)
+rsa_key_size=4096
+############# VARIABLES FOR POSSIBLE ADJUSTMENT ###########
+data_path="./certbot"
+email="lukasz.taraszka@revolve.pro"
```

Comment:
Jednak wolałbym, żeby ten mail był uzupełniany na serwerze dopiero, zaważywszy, że to jest opensource.

Reply:
wywaliłem

Reply:
dzięki ;)